src/metrics/loc.rs

```diff
@@ -480,28 +480,47 @@ mod tests {
 
         // LLOC returns three because there is an empty Rust statement
         check_metrics!(
-            "let a = 42;\n
-             if true {\n
-                42\n
-             } else {\n
-                43\n
+            "let a = 42;
+             if true {
+                42
+             } else {
+                43
              };\n",
             "foo.rs",
             RustParser,
             loc,
             [(lloc, 3, usize)]
         );
-    }
 
-    #[test]
-    fn test_lloc_c() {
         check_metrics!(
-            "for (;;)\n
+            "for (;;)
                 break;\n",
             "foo.c",
             CppParser,
             loc,
             [(lloc, 2, usize)]
         );
     }
+
+    #[test]
+    fn test_general_loc() {
+        check_metrics!(
+            "def func(a,
+                      b,
+                      c):
+                 print(a)
+                 print(b)
+                 print(c)\n",
+            "foo.py",
+            PythonParser,
+            loc,
+            [
+                (sloc, 6, usize),
```

Comment:
Please add a comment for each value to explain why the value (exactly as you did for Halstead).